### PR TITLE
Fixed typo CODE_OF_CONDUCT-ko.md (korean review requested)

### PR DESCRIPTION
**Pull request by rjsdn0124**
## What does this PR do?
Fixed wrong Korean typo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: docs/CODE_OF_CONDUCT-ko.md [hunk 1]
```diff
@@ -18,15 +18,15 @@
 * 부적절한 것으로 간주될 수 있는 다른 행위
 
 프로젝트 유지자는 이 행동 강령을 따르지 않은 댓글, 커밋, 코드, 위키 편집, 이슈와 그 외 다른 기여를
```
Comment (l2hyunwoo) [inline docs/CODE_OF_CONDUCT-ko.md:20]:
```suggestion
프로젝트 메인테이너는 이 행동 강령을 따르지 않은 댓글, 커밋, 코드, 위키 편집, 이슈와 그 외 다른 기여를
```

File: docs/CODE_OF_CONDUCT-ko.md [hunk 2]
```diff
@@ -18,15 +18,15 @@
...
-삭제, 수정 또는 거부할 권리와 책임이 있습니다. 또한, 부적당하거나 험악하거나 공격적이거나 해롭다고
+삭제, 수정 또는 거부할 권리와 책임이 있습니다. 또한, 부합하지 않거나 험악하거나 공격적이거나 해롭다고
```
Comment (l2hyunwoo) [inline docs/CODE_OF_CONDUCT-ko.md:21]:
```suggestion
삭제, 수정 또는 거부할 권리와 책임이 있습니다. 또한, 행동 강령 규약에 부합하지 않거나 험악하거나 공격적이거나 해롭다고
```

File: docs/CODE_OF_CONDUCT-ko.md [hunk 3]
```diff
@@ -18,15 +18,15 @@
...
 생각하는 다른 행동을 한 기여자를 일시적 또는 영구적으로 퇴장시킬 수 있습니다.
```
Comment (l2hyunwoo) [inline docs/CODE_OF_CONDUCT-ko.md:22]:
```suggestion
생각되는 행동을 한 기여자를 일시적 또는 영구적으로 퇴장시킬 수 있습니다.
```

File: docs/CODE_OF_CONDUCT-ko.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # 컨트리뷰터/기여자들의 행동 강령 규약
 
 이 프로젝트의 컨트리뷰터이자 메인테이너로서, 개방적이고 환영하는 커뮤니티를 육성하기 위해
-우리는 이슈리포팅, 기능 요청, 문서 업데이트, Pull request 또는 Patch 제출 및 기타 활동을 통해
+우리는 이슈 보고, 기능 요청, 문서 업데이트, Pull request 또는 Patch 제출 및 기타 활동을 통해
```
Comment (l2hyunwoo) [inline docs/CODE_OF_CONDUCT-ko.md:4]: I think we can keep this part as is

**Review by l2hyunwoo — COMMENTED** [review]
👍🏻

File: docs/CODE_OF_CONDUCT-ko.md (hunk 2 above)
Comment (Helloyunho) [reply docs/CODE_OF_CONDUCT-ko.md:21]: hmmm I kinda think this sounds too aggressive (just my thoughts)

**Comment by eshellman** [comment]
@rjsdn0124 please accept the review suggestions

File: docs/CODE_OF_CONDUCT-ko.md (hunk 2 above)
Comment (l2hyunwoo) [reply docs/CODE_OF_CONDUCT-ko.md:21]: @Helloyunho I think it’s adequate action for malicious contributors.

**Review by l2hyunwoo — APPROVED** [review]
👍🏻 LGTM

**Comment by eshellman** [comment]
excellent collaboration

**Pull request merged**